Documentation/design/building-your-csv.md

```diff
@@ -236,6 +236,13 @@ The metadata section contains general metadata around the name, version and othe
 
 **Selectors** (optional): A label selector to identify related resources. Set this to select on current labels applied to this CSV object (if applicable).
 
+**InstallModes**: A set of `InstallMode`s that tell OLM which `OperatorGroup`s an Operator can belong to. An `InstallMode` consists of an `InstallModeType` and a boolean `Supported` flag. There are four `InstallModeTypes`: 
```

Comment:
I think this should mention the specific way that an operator "supports" an InstallMode - which is that it understands the OLM-written annotations on the pods